save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/DemoRunComponent.kt

```diff
@@ -37,9 +37,7 @@ import web.html.InputType
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 
-val diktatDemoComponent = diktatDemoComponent()
-
-private val diktatDemoDefaultCode = """
+private val defaultCode = """
```

Comment:
Not sure if we want to display this code as default. Probably should let demo owner upload file with name `inputFileName` in order to set default code. (should be done later)

Reply:
I'm not sure -- I think we should leave it as empty (when it's be ready for production) or have a map where key is language

Reply:
And we also should think of cookies, just as you suggested

Reply:
> or have a map where key is language
The point is that we do not what language is connected with the demo. The only thing that could tell us about the language is `inputFileName` extension which is not that reliable

Reply:
> > or have a map where key is language
> 
> The point is that we do not what language is connected with the demo. The only thing that could tell us about the language is `inputFileName` extension which is not that reliable

then -- empty by default

Reply:
> And we also should think of cookies, just as you suggested

agree, we should in separate tasks: https://github.com/saveourtool/save-cloud/issues/1864

Reply:
good